src/work-allocation-2/containers/work-case-list-wrapper/work-case-list-wrapper.component.ts

```diff
@@ -0,0 +1,275 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { AlertService, LoadingService } from '@hmcts/ccd-case-ui-toolkit';
+
+import { Caseworker } from 'api/workAllocation2/interfaces/common';
```

Comment:
Coming from `api/`, which it shouldn't.

Reply:
Now added a common interfaces to the src instead of referencing the api.